app/src/main/java/io/apicurio/registry/rules/compatibility/AvroCompatibilityChecker.java

```diff
@@ -54,9 +54,9 @@ public boolean isCompatibleWith(CompatibilityLevel compatibilityLevel, List<Stri
 
         try {
             schemaValidator.validate(toValidate, existingSchemas);
-            return true;
+            return new CompatibilityExecutionResult(true, Collections.emptySet());
         } catch (SchemaValidationException e) {
-            return false;
+            return new CompatibilityExecutionResult(false, Collections.emptySet());
```

Comment:
I looked, and sadly there is no additional information provided by the Avro compatibility validators.  :(